test/scripts/readme-examples-test.sh

```diff
@@ -0,0 +1,41 @@
+./lost pipeline \
+  --generate 1 \
+  --plot-raw-input raw-input.png \
+  --plot-input input.png
+./lost pipeline \
+  --generate 1 \
+  --generate-x-resolution 1024 \
+  --generate-y-resolution 1024 \
+  --fov 30 \
+  --generate-reference-brightness 100 \
+  --generate-spread-stddev 1 \
+  --generate-read-noise-stddev 0.05 \
+  --generate-ra 88 \
+  --generate-de 7 \
+  --generate-roll 0 \
+  --plot-raw-input raw-input.png \
+  --plot-input annotated-input.png
+
+wget https://markasoftware.com/img_7660.png
+
+./lost database \
+  --max-stars 5000 \
+  --kvector \
+  --kvector-min-distance 0.2 \
+  --kvector-max-distance 15 \
+  --kvector-distance-bins 10000 \
+  --output my-database.dat
+./lost pipeline \
```

Comment:
Right now this test doesn't verify anything. Either add a check that the result is correct or delete the whole test.

Reply:
The point was to test if it ran at all. The other tests were there to test accuracy, this was just to make sure things would build correctly